test/NeinLinq.Fakes/InjectableQuery/FunctionsBase.cs

```diff
@@ -83,5 +84,16 @@ public double VelocityWithNonPublicSibling(Dummy value)
         {
             throw new NotSupportedException();
         }
+
+        [InjectLambda]
+        public double VelocityWithHiddenSibling(Dummy value)
+        {
+            throw new NotSupportedException();
+        }
+
+        public Expression<Func<Dummy, double>> VelocityWithHiddenSibling()
+        {
+            return v => v.Distance / v.Time;
+        }
     }
```

Comment:
You could include a test for virtual members, to prove, that those are working also:

```C#
        [InjectLambda]
        public double VelocityWithVirtualSibling(Dummy value)
        {
            throw new NotSupportedException();
        }

        public abstract Expression<Func<Dummy, double>> VelocityWithVirtualSibling();
```

Continued in `ConcreteFunctions`